main.go

```diff
@@ -289,43 +287,24 @@ func getTLSCertificate() (tls.Certificate, error) {
 func getFixtures(fixturesPath string) (*spec.Fixtures, error) {
 	var data []byte
 	var err error
-	var isYAML bool
 
 	if fixturesPath == "" {
 		// And do the same for fixtures
 		data, err = Asset("openapi/openapi/fixtures3.json")
 	} else {
 		data, err = ioutil.ReadFile(fixturesPath)
-
-		if filepath.Ext(fixturesPath) == ".yaml" {
-			isYAML = true
-		}
```

Comment:
is it worth returning a friendly error if the extension is not json in case people were using yaml before?

Reply:
Good call. Added an additional check in 16d5511.